README.md

```diff
@@ -7,7 +7,7 @@ This package is not yet published on PyPi, but can be installed from the local r
 
     pip install .
 
-To create a new frontend session which is controlled by the wrapper instead of connecting to an existing frontend session, you also need to install the `selenium` Python library. You also need to make sure that your desired browser is installed, together with a corresponding web driver.
+To create a new frontend session which is controlled by the wrapper instead of connecting to an existing frontend session, you also need to install the `selenium 4.3+` Python library. You also need to make sure that your desired browser is installed, together with a corresponding web driver.
```

Comment:
I checked the Selenium Python source, and it looks like [this API has been available for the past 14 years](https://github.com/SeleniumHQ/selenium/commit/d2ffd4b185e6d9d24cfe4051c0723c03a8359606) (it's just the deprecation that's new). So we don't need to require a newer Selenium version here (this code works fine on my currently installed 3.141.0), and we don't need to do anything to support the old API (if we did, we could have done an import test like the one for `StrEnum`).

The actual code change looks good to me.

Reply:
It's good news that we don't have to worry about backward compatibility.